ceph-releases/luminous/centos/7/daemon/clean_container.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+INITIAL_SIZE=$(du -sm / 2>/dev/null | awk {'print $1}')
+DEBUG=
+
+PURGES="/tmp/* /var/tmp/* /usr/lib/{dracut,locale,systemd,udev} /usr/bin/hyperkube /usr/bin/etcd /usr/bin/systemd-analyze /etc/{udev,selinux} /usr/lib/{udev,systemd}"
+PURGES="$PURGES /usr/share/hwdata/{iab.txt,oui.txt}"
+rm -rf $PURGES
+
+PURGE_PKGS="groff-base e2fsprogs sharutils shadow-utils yum-utils python-gobject-base python-kitchen passwd bind-license policycoreutils-python epel-release"
+PURGE_PKGS="$PURGE_PKGS libselinux-python libss libcgroup dbus-python yum-plugin-ovl rbd-mirror libxml2-python python-IPy checkpolicy libsemanage-python setools-libs"
+PURGE_PKGS="$PURGE_PKGS systemd-sysv dbus-glib gobject-introspection audit-libs-python"
+
+# Step 1
+# Let's remove easy stuff
+strip -s /usr/local/bin/{confd,forego}
+
+# Compressing those very big binaries
+# As we don't run them often, the peformance penalty isn't huge
+for binary in /usr/bin/{etcdctl,kubectl} /usr/local/bin/{forego,confd}; do
+   gzexe $binary && rm ${binary}~
```

Comment:
indent 2

Reply:
I fixed all the indent 2 issues and refactored the Dockerfile to get that very long RUN entry much more readable.